ol-keycloak/ol-spi/src/main/java/forms/login/freemarker/OlFreeMarkerLoginFormsProvider.java

```diff
@@ -36,16 +36,16 @@ public OlFreeMarkerLoginFormsProvider(KeycloakSession session) {
     protected void createCommonAttributes(Theme theme, Locale locale, Properties messagesBundle, UriBuilder baseUriBuilder, LoginFormsPages page) {
         super.createCommonAttributes(theme, locale, messagesBundle,baseUriBuilder,page);
         if (page == LOGIN) {
-            String firstName = "";
+            String attemptedName = "";
             Boolean hasCredentials = false;
             UserModel user = context.getUser();
             if (context.getUser() !=  null) {
-                firstName = user.getFirstName();
+                attemptedName = user.getFirstName().concat(" ").concat(user.getLastName());
```

Comment:
This line throws an error when the first name is null.  I'm not sure why that would occur in the real world, but I also don't know how we can be sure it never happens.